### PR TITLE
doc-data-quality: use base form component for text input

### LINKED ISSUE #19: Step 4: Documentation and data quality
**Changes**
Check Forms field limit of character and change so text typo correction

**Additional notes**
List any notes you collected that support the original task and include a screenshot if applicable.

**Assumptions**
List any assumptions that you are aware of but still need to be reviewed.

**Acceptance Criteria**
Verify and adjust the character limit for form fields and correct any text typos.

**Risks? (optional)**
Potential risks and how to best mitigate them.

### PR DESCRIPTION
**Pull request by rekt-hard**
## Description

Bugfix

#### What does this PR do?

Comment out two fields, for which we do not have a backend pendant at the moment. It requires custom fields to be implemented.
Use base components and set attributes correctly.

Current state:
image

### Checks

- [ ] Summary updated
- [ ] Version view updated
- [ ] Documentation added
- [ ] Tests added
- [ ] E2e tests created
- [ ] Successfully ran e2e tests before merge

closes #19

### CODE REVIEW

File: libs/damap/src/lib/components/dmp/doc-data-quality/doc-data-quality.component.html [hunk 1]
```diff
@@ -74,10 +66,11 @@
     }}</mat-label>
     <input
       matInput
-      #input
+      #inputMetadata
+      [formControl]="metadata"
       maxlength="4000"
       placeholder=" As there are no domain specific metadata standards applicable, we will provide a README file with an explanation of all values and terms used [at file level/at dataset level/at project level]" />
-    <mat-hint align="end">{{ input.value.length }}/4000</mat-hint>
+    <mat-hint align="end">{{ inputMetadata.value.length }}/4000</mat-hint>
```
Comment (rekt-hard) [inline libs/damap/src/lib/components/dmp/doc-data-quality/doc-data-quality.component.html:73]: Would be nice to have a reusable component or integrate it with the base [`app-textarea-wrapper`](https://github.com/tuwien-csd/damap-frontend/tree/5d7d72be6ffc0b7e93db6977bbd1e9d87972e8f2/libs/damap/src/lib/shared/textarea-wrapper) or [`app-input-wrapper`](https://github.com/tuwien-csd/damap-frontend/blob/5d7d72be6ffc0b7e93db6977bbd1e9d87972e8f2/libs/damap/src/lib/shared/input-wrapper/) as is done here:
https://github.com/tuwien-csd/damap-frontend/blob/5d7d72be6ffc0b7e93db6977bbd1e9d87972e8f2/libs/damap/src/lib/shared/input-wrapper/input-wrapper.component.html#L2-L5

Comment (rekt-hard) [reply libs/damap/src/lib/components/dmp/doc-data-quality/doc-data-quality.component.html:73]: Wrong line. Should have been the tooltip :)

**Comment by rekt-hard** [comment]
For the tooltip inside a form, would be nice to have a reusable component or integrate it with the base [`app-textarea-wrapper`](https://github.com/tuwien-csd/damap-frontend/tree/5d7d72be6ffc0b7e93db6977bbd1e9d87972e8f2/libs/damap/src/lib/shared/textarea-wrapper) or [`app-input-wrapper`](https://github.com/tuwien-csd/damap-frontend/blob/5d7d72be6ffc0b7e93db6977bbd1e9d87972e8f2/libs/damap/src/lib/shared/input-wrapper/) as is done here:
https://github.com/tuwien-csd/damap-frontend/blob/5d7d72be6ffc0b7e93db6977bbd1e9d87972e8f2/libs/damap/src/lib/shared/input-wrapper/input-wrapper.component.html#L2-L5

**Review by lpandath — APPROVED** [review]
Looks good! For now only the base question in place then.

**Pull request merged**